src/app/projects/projects-list/projects-list.component.html

```diff
@@ -122,6 +124,11 @@
             <p *ngIf="!project.technologies.length">-</p>
           </ul>
         </td>
+
+        <td class="vf-table__cell">
+          {{project.cellCount}}
```

Comment:
Actually, if there is no cell count we should display "-" as with the others

Reply:
Or `N/A`

Reply:
I used "-" so that it's consistent with other columns